storage/cloud-client/notification_polling_test.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2017 Google Inc. All rights reserved.
```

Comment:
In general unit tests are completely useless for the purpose of testing samples. We generally write system tests. I'll link you to a few internal guides.